docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -491,10 +487,7 @@ export default class MethodPage extends React.PureComponent<Props, State> {
       exampleHeaders,
       debugRequest: urlDebugRequest || method.exampleRequests[0] || '',
       debugResponse: '',
-      additionalHeaders:
-        urlHeaders ||
-        stateHeaders ||
-        (exampleHeaders.length === 1 ? exampleHeaders[0].value || '' : ''),
```

Comment:
I'm not sure but isn't this okay to let as it is?

Reply:
Reverted it. Thanks~!